lnrpc/walletrpc/walletkit.pb.go

```diff
@@ -147,22 +190,40 @@ var (
 		11: "WITNESS_KEY_HASH",
 		12: "NESTED_WITNESS_KEY_HASH",
 		13: "COMMITMENT_ANCHOR",
+		14: "COMMITMENT_NO_DELAY_TWEAKLESS",
+		15: "COMMITMENT_TO_REMOTE_CONFIRMED",
+		16: "HTLC_OFFERED_TIMEOUT_SECOND_LEVEL_INPUT_CONFIRMED",
+		17: "HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL_INPUT_CONFIRMED",
+		18: "LEASE_COMMITMENT_TIME_LOCK",
+		19: "LEASE_COMMITMENT_TO_REMOTE_CONFIRMED",
+		20: "LEASE_HTLC_OFFERED_TIMEOUT_SECOND_LEVEL",
+		21: "LEASE_HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL",
+		22: "TAPROOT_PUB_KEY_SPEND",
 	}
 	WitnessType_value = map[string]int32{
-		"UNKNOWN_WITNESS":                    0,
-		"COMMITMENT_TIME_LOCK":               1,
-		"COMMITMENT_NO_DELAY":                2,
-		"COMMITMENT_REVOKE":                  3,
-		"HTLC_OFFERED_REVOKE":                4,
-		"HTLC_ACCEPTED_REVOKE":               5,
-		"HTLC_OFFERED_TIMEOUT_SECOND_LEVEL":  6,
-		"HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL": 7,
-		"HTLC_OFFERED_REMOTE_TIMEOUT":        8,
-		"HTLC_ACCEPTED_REMOTE_SUCCESS":       9,
-		"HTLC_SECOND_LEVEL_REVOKE":           10,
-		"WITNESS_KEY_HASH":                   11,
-		"NESTED_WITNESS_KEY_HASH":            12,
-		"COMMITMENT_ANCHOR":                  13,
+		"UNKNOWN_WITNESS":                                    0,
```

Comment:
Couldn't you iterate through this array?

Reply:
> Couldn't you iterate through this map?

This would only test that `StandardWitnessType` contains all the enum values defined in the protobuf enum. But it doesn't test the reverse: that the protobuf enum contains all the enum values defined in `StandardWitnessType`.

The reason these two enums went out of sync is because `StandardWitnessType` was added to, but the protobuf wasn't updated. So the test as it is ensures that for every enum value of  `StandardWitnessType`, (or at least every enum value with a string representation), the corresponding enum value exists in the protobuf definition.